### ISSUE TITLE
req.options: what options are available?

### ISSUE THREAD

**Issue by shaunc** [balderdashy/sails-docs#505]
The `req.options` doc states: "Additionally, several special req.options objects are available for use with blueprints,...". I noticed that [sails-generate-ember-blueprints](https://github.com/mphasize/sails-generate-ember-blueprints) uses req.options.associations, which I might be interested in using to [include toMany relations](http://stackoverflow.com/questions/31285156/including-tomany-list-of-ids-in-rest-response-from-sailsjs).

Could we get some doc on these (or at least a pointer to the code where they are filled in)?

**Comment by sailsbot** [comment]
Thanks for posting, @shaunc.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
